.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.{erl,hrl}]
+indent_style = space
```

Comment:
@eproxus I tested indentation in aect_dispatch.erl (that has more than one line with tab as indentation) - amending one does not change all lines (tested emacs and intellij idea).